src/go/k8s/pkg/resources/configmap.go

```diff
@@ -224,18 +224,34 @@ func (r *ConfigMapResource) CreateConfiguration(
 	cr := &cfg.NodeConfiguration.Redpanda
 
 	internalListener := r.pandaCluster.InternalListener()
+	var internalAuthN *string
```

Comment:
can we add something to configmap_test for these?

Reply:
Ok I will add